sql-plugin/src/main/scala/com/nvidia/spark/rapids/VersionUtils.scala

```diff
@@ -24,6 +24,8 @@ object VersionUtils {
 
   lazy val isSpark320OrLater: Boolean = cmpSparkVersion(3, 2, 0) >= 0
 
+  lazy val isSpark311Before: Boolean = !isSpark311OrLater
```

Comment:
This should be named `isPriorToSpark311` or `isBeforeSpark311` if we keep it, otherwise one could misread that as "is Spark 3.1.1 before the current version."  However I would argue it's not necessary.  IMHO `!isSpark311OrLater` is pretty clear at the call site.  Otherwise one would argue we need equivalent "before" vals for all the other Spark versions.